data/pages-content/virtual-swag.md

```diff
@@ -1,152 +1,82 @@
-  <div>
-    <img src="/swag/Bloomberg_swag.png" width="4800 " height="2700" />
+<a className="img" id="sentry" target="_blank" href="https://sentry.io/signup">
+  <div className="logo" style={{marginBottom: 15}}>
+    <img src="/swag/sentry.svg" width="500" height="119" />
   </div>
 </a>
 
-<a href="/swag/Bloomberg_swag.pdf" target="_blank">Get your swag here</a>
+  <img src="/swag/sentry-swag-0.png" width="700" height="700" />
+  [Get your swag here](https://sentry.io/signup)
 
 ---
 
-# Numberly
-
 <a className="img" id="numberly" target="_blank" href="https://numberly.com/en/technology/">
-  <div className="logo">
-    <img src="/swag/numberly.svg" width="469" height="119" />
+  <div className="logo" style={{marginBottom: 15}}>
+    <img src="/swag/numberly.svg" width="500" height="119" />
   </div>
 </a>
 
-<a className="img" target="_blank" href="/swag/Numberly_swag.pdf">
-  <img src="/swag/Numberly_swag.png" width="2480" height="3508" />
-</a>
-
-<a href="/swag/Numberly_swag.pdf" target="_blank">Get your swag here</a>
-
+<div style={{marginBottom: 15}}>
+  <a className="img" target="_blank" href="/swag/numberly-swag.pdf">
+    <img src="/swag/numberly-swag-0.png" width="700" height="300" />
+  </a>
+</div>
+[Check out the opportunities here](/swag/numberly-swag.pdf)
 
 ---
 
-# Optum
-
-<a className="img" id="optum" target="_blank" href="https://optum.co/joinus">
-  <div className="logo">
-    <img src="/swag/optum.svg" width="708" height="205" />
+<a className="img" id="Google Cloud" target="_blank" href="https://goo.gle/europython23">
+  <div className="logo" style={{marginBottom: 15}}>
+    <img src="/swag/googlecloud.svg" width="500" height="119" />
   </div>
 </a>
-<a className="img" target="_blank" href="/swag/Optum_swag.pdf">
-  <div>
-    <img src="/swag/Optum_swag.png" width="3508" height="2480" />
-  </div>
-</a>
-
-<a href="/swag/Optum_swag.pdf" target="_blank">Get your swag here</a>
 
+<div style={{marginBottom: 15}}>
+  <a className="img" target="_blank" href="/swag/google-cloud-swag.pdf">
+    <img src="/swag/google-cloud-swag.png" width="700" height="300" />
+  </a>
+</div>
+[Get your swag here](https://goo.gle/europython23)
 
 ---
-
-# Google Cloud
-
-<a className="img" id="Google Cloud" target="_blank" href="https://cloud.google.com/developers/europython">
-  <div className="logo">
-    <img src="/swag/googlecloud.svg" width="2899" height="512" />
+<a className="img" id="manning" target="_blank" href="http://mng.bz/vnWa">
+  <div className="logo" style={{marginBottom: 15}}>
+    <img src="/swag/manning.svg" width="500" height="119" />
   </div>
 </a>
 
-<br></br>
-<a href="https://cloud.google.com/developers/europython/" target="_blank">Get your swag here</a>
+Get 35% discount for all Manning products in all formats using the link below:
 
+<div style={{marginBottom: 15}}>
+    <img src="/swag/manning-swag.png" width="400" height="300" />
+</div>
+[Get your swag here](http://mng.bz/vnWa)
```

Comment:
I think, this link is not yet active

Reply:
Hmmm you are right. I've now removed the QR code because it doesn't do anything and we've already linked it from the swag link below. But I've now added the discount code which seems to work.

Reply:
ah, I see. The code works on the checkout. Somehow I thought that it will display all books with the discount already. Maybe we can clarify this saying smth. like: "please use this promo code during checkout to receive a 35% discount on all products" 🙃

Reply:
If QR code is not used, we can remove it from `public/swag/manning-swag.png`

Reply:
Done and don! Thanks for keeping us clean!